app/views/users/index.html.erb

```diff
@@ -1 +1,8 @@
-<h1>Here is a list of all users</h1>
\ No newline at end of file
+<h1>Here is a list of all users</h1>
+<% @users.each do |user| %>
+  <div class="users-container">
+    <img class="user-photo" src="<%= user.photo %>"><br>
+    <h2><%= user.name %></h2><br>
+    <p class="text-usercard-index">Number of Posts: <%= user.posts_counter %></p><br><br>
+  </div>
+<% end %>
```

Comment:
- [ ] Hello @Felipeg005. Let's look back at the previous **`required change`** comment. The problem the previous reviewer encountered is that one cannot click on a `user` in the `index page` and be directed to the `show page`.

   Think about a normal user of your app. Should we expect the user to manually type the above paths that you have provided in order to use the app? The logical thing would be to create links that can be clicked to direct us to those other parts of the app. Is that right?

    One of these properties should be clickable to direct the user to the `show page`. Either the image, name or the whole **`<div>`**
    
    To ensure that we implement the previously requested change please create a:
    - Link to the user show page.